nml/actions/action0properties.py

```diff
@@ -849,7 +871,9 @@ def check_accept(acp):
     'prod_increase_msg'      : {'size': 2, 'num': 0x0D, 'string': 0xDC},
     'prod_decrease_msg'      : {'size': 2, 'num': 0x0E, 'string': 0xDC},
     'fund_cost_multiplier'   : {'size': 1, 'num': 0x0F},
-    'cargo_types'            : {'custom_function': industry_cargo_types}, # = prop 25+26+27+28 combined in one structure
+    'prod_cargo_types'       : {'size': 2, 'num': 0x10, 'value_function': lambda value: cargo_list(value, 2), 'warning': "Please consider replacing 'prod_cargo_types' with 'cargo_types'"},
```

Comment:
I think copy-pasting all these warnings is a bit ugly. Something like `{..., 'replaced_by': "cargo_types"}` would be better, but `parse_var()` doesn't know the current name...

Fixing the TODO at action2var.py:951 would allow this, I will try to do that.

Reply:
Yeah for properties, name is known so it should be easy to add `replaced_by`. But for variables it's less easy.